### PR TITLE
Vertical navigation support dynamic translations and badge counts

### PR DESCRIPTION
**Pull request by chalettu**
This enables the page to be refreshed without doing a window refresh.  
PV https://www.pivotaltracker.com/story/show/134436111

### CODE REVIEW

**Comment by chalettu** [comment]
@miq-bot add_label bug

**Comment by chalettu** [comment]
Ultimately the reason we had issues translating is that we changed the source title's language in our navigation provider, since the provider stayed in its current state after a state reload, the system attempted to translate the translated title.  Solution was to create a variable of the original untranslated string that we can then use in our navigation controller to translate.

**Comment by chalettu** [comment]
FYI - Only our Dashboard title changes at the moment.  We need these other terms to be translated in our menu.  @chriskacerguis where do I file a ticket requesting translations?

**Comment by chriskacerguis** [comment]
@chalettu in BZ, set the "Component" to "Internationalization" (sp?).

File: client/app/core/navigation.config.js [hunk 1]
```diff
@@ -3,42 +3,43 @@
...
 export function navConfig(NavigationProvider) {
   const dashboard = createItem(
-    N_('Dashboard'),
+    'Dashboard',
     'dashboard',
     'fa fa-dashboard'
   );
+
   const services = createItem(
-    N_('Services'),
```
Comment (chriskacerguis) [inline client/app/core/navigation.config.js]: Do we need the N_ for the translations (or did we change something I missed)?

Comment (chalettu) [inline client/app/core/navigation.config.js]: Translation happens in the navigation controller.  It was already doing it there as well.  It is better in this instance to do it in the navigation controller.

**Comment by chalettu** [comment]
dmo

**Pull request merged**